docs/data-apps/attribution-modeling/index.md

```diff
@@ -0,0 +1,73 @@
+
+## Preparing Views for the Dashboards
+Note that all these settings are global for all users, meaning if you change them they will be changed for everyone. The first user of the app will have to define at least one `View` which is the dataset needed to generate the charts. Defining a View can be done on the `Settings` page.
+
+### Defining a Last N Days (auto-refreshed) View
+The so-called `Default` view is to be used for generating a dataset that has a rolling conversion window of last nth day and will be refreshed automatically. The app will save the last-refreshed date with the View configurations and any subsequent day a user logs back in the app, a query will run in the background to look for any newly processed conversion event in the conversion source and if there is, the default dataset is refreshed by running all the queries that are needed to generate data for the charts to populate. Once the update finishes the conversion window should display the new date range.
+
+#### Basic setup:
+1. Set the `auto-update days`: the number of days since the last conversion event defined here will define the conversion window.
+2. Set a `currency symbol` (defaults to £)
+3. Decide if you would like to `use Non-Conversions`:
+Use this with caution, currently it uses the `snowplow_attribution_paths_to_non_conversion` table as is without considering the conversion period. The intention is to make this a fully automated feature in the near future so watch for updates on this.
+
+#### Connect your Data Sources:
+1. Select your schema that contains the derived unified and attribution tables: this will trigger an update which checks for any tables with the names closest to what the app expects.
+2. After waiting for the update to take place you can revise if the auto-detected source tables are in line with your expectations, you can change them to any other existing tables you have in case they are not correct.
+3. Overwrite the attribution_manifest table. Most likely the schema name will have to be modified. Please keep the `schema_name.table_name` notation here. Make sure you press enter once modified.
+4. (Optional but recommended) Specify the Spend Source: this will most likely be a view you created on top of your table that holds your marketing spend data. The view should make sure you align the expected field names. It should have `campaign`, `channek`, `spend` and `spend_tstamp` for the analysis to work. Doing this will make sure you have return on advertising spend (ROAS) calculation in your overview. Make sure you press enter once modified.
+5. Once happy with all the imputs press Create View button. It will first run a validation against the data sources making sure it has all the fields it needs. After that it will run the queries that generate the data necessary to populate the dashboards. They will be saved as csv files that app will read from when selecting the View on the sidebar.
+
```

Comment:
Can you make this part first/at a different level - I'm personally not a fan of docs that have instructions for one method and then the next method says something like "use most of the steps above but...". It can lead people to skip over this then have to go back and work out what is/isn't relevant.